resumebuilder/src/components/Basics.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const ResumeBasics = props => {
+	const { applicant } = props;
+
+	return (
+		<div>
+			<h4>Basics</h4>
+
+			<ul>
+				<li>{applicant.basics.name}</li>
+				<li>{applicant.basics.label}</li>
+				<li>{applicant.basics.email}</li>
+				<li>{applicant.basics.phone}</li>
+				<li>{applicant.basics.website}</li>
+			</ul>
+			<article>{applicant.basics.summary}</article>
+		</div>
+	);
+};
```

Comment:
@7FigureSwagger you can install Prettier and configure your VSCode to auto-format on save your code.

I'll leave this link so you can do it:
https://www.codereadability.com/automated-code-formatting-with-prettier/

Reply:
thanks! I actually use and love prettier! I just don't have it set to auto format haha. Will watch that in the future

Reply:
Oh that's nice to hear, auto format will change your life  😎